prover_rust/src/version.rs

```diff
@@ -0,0 +1,19 @@
+use std::cell::OnceCell;
+
+static DEFAULT_COMMIT: &str = "unknown";
+static mut VERSION: OnceCell<String> = OnceCell::new();
+
+pub const TAG: &str = "v4.4.3";
```

Comment:
Do we need to begin with `v4.4.3` (aligning scroll monorepo prover's version) for convenience? cc @HAOYUatHZ

Reply:
Now the tag number will be tracked to the common/version/version.go and set when running cargo build.
See the Makefile and check the GO_TAG

Reply:
got it.